client/account/AccountRoute.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from 'react';
+
+import { useRouteParameter, useRoute } from '../contexts/RouterContext';
+import { SideNav } from '../../app/ui-utils';
+import AccountProfilePage from './AccountProfilePage';
+import './sidebarItems';
+
+const AccountRoute = () => {
+	const page = useRouteParameter('group');
+	const router = useRoute('account');
+
+	useEffect(() => { !page && router.push({ group: 'profile' }); }, [page, router]);
+
+	useEffect(() => {
+		SideNav.setFlex('accountFlex');
+		SideNav.openFlex();
+	});
+
+	return <>
+		{{
+			profile: <AccountProfilePage />,
+		}[page]}
+	</>;
```

Comment:
The problem with this approach is that, besides being somewhat hard to read, the page elements are instantiated regardless of the `page` value. It's weird, but a simple

```js
if (page === 'profile') {
	return <AccountProfilePage />;
}

return null;
```

is way better to read and maintain.